content/format-reduction/index.md

```diff
@@ -0,0 +1,56 @@
+---
+rfc:
+start_date: 2018-07-26
+pr:
+status: draft
+---
+
+# Format reduction
+
+## Summary
+
+This RFC proposes to reduce the amount of representations in Registers.
+
+
+## Motivation
+
+The Registers data model and its REST API is still in motion. The amount of
+formats suggested in the specification and implemented in the reference
+implementation increase unnecessarily the complexity of evolving them. For
+example, we want to evolve who the schema fits into the data model and how it
```

Comment:
Maybe it's just me but I'm not totally clear on how this follows on logically from the previous sentence. In this sentence:

> ... we want to evolve who the schema fits into the data model and how it is surfaced to the user

does _it_ refer to the schema?

Reply:
yes, _it_ refers to the schema.

Reply:
Are "schema" and "data model" defined terms in the spec right now? If not it may be worth clarifying. It makes sense to me right now, but I'm not sure how much knowledge we should assume in these RFCs.